libs/langchain-google-common/src/types.ts

```diff
@@ -309,6 +309,8 @@ export interface GeminiContent {
 
 export interface GeminiTool {
   functionDeclarations?: GeminiFunctionDeclaration[];
+  googleSearchRetrieval?: object;
+  retrieval?: object;
```

Comment:
I would have like to see these as actual types or classes rather than as `object`, but I don't think this should hold off on merging.